task/mock/scheduler.go

```diff
@@ -22,28 +19,19 @@ type Scheduler struct {
 
 	lastTick int64
 
-	claims map[string]*Task
-	meta   map[string]backend.StoreTaskMeta
+	claims map[string]*platform.Task
```

Comment:
Since we're in this code anyway, can you change this to `map[influxdb.ID]*platform.Task`?